client/my-sites/checkout/checkout-thank-you/domains/style.scss

```diff
@@ -2,26 +2,89 @@
 @import '@wordpress/base-styles/breakpoints';
 @import '@wordpress/base-styles/mixins';
 
-.is-section-checkout-thank-you .layout__content {
-    padding-right: 0;
-    padding-left: 0;
+.is-section-checkout-thank-you {
+	background: var( --studio-white );
+	
+	.layout__content {
+		padding-right: 0;
+		padding-left: 0;
+	}
```

Comment:
Does it make more sense to move this chunk of CSS into the core thank you component styles? We're going to want this across all of the post-checkout thank you pages, not just the domains one.